hypothesis-python/tests/cover/test_charmap.py

```diff
@@ -169,3 +170,15 @@ def test_regenerate_broken_charmap_file():
 
 def test_exclude_characters_are_included_in_key():
     assert cm.query() != cm.query(exclude_characters='0')
+
+
+def test_error_writing_charmap_file_is_suppressed(monkeypatch):
+    def broken_mkstemp(dir):
+        raise RuntimeError()
+
+    monkeypatch.setattr(tempfile, 'mkstemp', broken_mkstemp)
+
+    cm._charmap = None
+    os.unlink(cm.charmap_file())
```

Comment:
This will cause a nasty performance hit next time someone needs the charmap - can you do `saved = cm._charmap; cm._charmap = None; <unlink>; cm._charmap = saved` to avoid this?

Reply:
Will do.